subcommands/waves/init.go

```diff
@@ -145,6 +150,31 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(api.FactoryCreateWave(factory, &wave), "Failed to create a wave")
 	}
+
+	if !hasStaticDelta(new_targets) {
+
+		fmt.Print(`
+WARNING: You created a wave for a target version without static deltas.
+
+We recommend that you generate static deltas for your production targets to optimize
+OTA update download. Please, consider generating a static delta for targets using:
+$ fioctl targets static-deltas
+
+You can then cancel this wave and create a new one for a target with a static delta.
```

Comment:
```suggestion
You can then cancel the wave and create a new one by using a static delta.
```

Optional suggestion! Just shortening the sentence a bit.

Reply:
@kprosise @camilamacedo86 This suggestion makes a message somewhat misleading.

A user cannot create a wave _by using a static delta_, but a user can create a wave _for a target which has static deltas defined_.

So, I suggest to either revert this message to its original, or make a new suggestion which provides a more accurate context to the user.

Reply:
I agree. Good point.